docs/src/pages/[platform]/components/phonenumberfield/react.mdx

```diff
@@ -44,21 +44,26 @@ which will auto-populate the country or region code select field.
   </ExampleCode>
 </Example>
 
-### Country or Region Code Select Properties
+### Dial Code Select Properties
 
-The country or region code selector can be customized by setting several properties when using the `PhoneNumberField` primitive. The custom
-properties specific to the country or region code selector are the following:
+<Alert variation="info">All `countryCode` fields are being deprecated in favor of the new `dialCode` fields.  If you still have instances of `defaultCountryCode`,
```

Comment:
```suggestion
<Alert variation="info"  role="none">All `countryCode` fields are being deprecated in favor of the new `dialCode` fields.  If you still have instances of `defaultCountryCode`,
```

Tiny nit: can we add this role="none" when we use Alerts on the docs so SRs don't announce them when Next loads the page chunk?